metadata-ingestion/src/datahub/ingestion/source/powerbi/powerbi.py

```diff
@@ -422,6 +443,16 @@ def chart_custom_properties(dashboard: PowerBiAPI.Dashboard) -> dict:
         if owner_mcp is not None:
             list_of_mcps.append(owner_mcp)
 
+        if self.__config.extract_endorsements_to_tags and dashboard.tags:
```

Comment:
Could we go for the append method? where append the MCP if flag is enabled to avoid checking at multiple places

Reply:
Meaning something like the `tags` in the dataclasses are populated only when `extract_endorsements_to_tags=True`? 

Then the code would look something like this, right?
`
for tag in dashboard.tags:
    list_of_mcps.append(
        self.new_mcp(
            Constant.DASHBOARD,
            dashboard_urn,
            Constant.GLOBAL_TAGS,
            self.transform_tags(dashboard.tags)
            tags,
        )
    )
`